tests/oesign-test-engine/expected_invalid_parameter_op1.txt

```diff
@@ -0,0 +1,2 @@
+*** ERROR: oe_sgx_sign_enclave_from_engine() failed: result=OE_INVALID_PARAMETER (0x3)
+/home/sagoel/mystikos/build/bin/myst: error: Failed to sign "hello.signed/lib/openenclave/mystenc.so"
```

Comment:
Maybe remove your alias?

Reply:
Oops! I missed that. Thank you Radhika!

Reply:
Ah thank you for the comment, I was blind to that. Will fing another way to verify output.